lib_expense/src/test/java/com/example/expenselogger/lib_expense/domain/usecase/DeleteExpenseTest.kt

```diff
@@ -23,4 +26,13 @@ public class DeleteExpenseTest {
             deleteExpense.invoke(expense)
             assertThat(expenseContract.getExpenses().size).isEqualTo(0)
         }
+
+    @Test
+    public fun `verify that deleteExpense usecase calls expenseContract`(): Unit = runBlockingTest {
+        val expense = DummyData.expense
+        val mockExpenseContract: ExpenseContract = mock(ExpenseContract::class.java)
+        val mockDeleteExpense = DeleteExpense(mockExpenseContract, TestAsyncExecutor())
```

Comment:
Rename this variable. It is not a mocked object

Reply:
I will rename it; I wanted to distinguish it from the member variable. Also why I prefixed "mock" to the other usecase variables

Reply:
OK. You need to make the tests uniform. You either choose to mock deps or use fakes

Reply:
Okay. I will use mocks all through the test class